openedx/core/djangoapps/content_staging/views.py

```diff
@@ -182,16 +182,16 @@ def _save_static_assets_to_clipboard(
         Helper method for "post to clipboard" API endpoint. This deals with copying static files into the clipboard.
         """
         files_to_save: list[StagedContentFileData] = []
-        for f in static_files:
+        for sf in static_files:
```

Comment:
I had to rename this variable because the `f` used as an iterator in this for loop is considered the same as the `f` used as the iterator in the second for loop in this function. I don't like how python does scoping.